workloads/virtual-machines/disks-and-volumes.md

```diff
@@ -373,3 +375,124 @@ spec:
 
 Ephemeral VMs very often come with read-only root images and limited tmpfs space. In many cases this is not enough to install application dependencies and provide enough disk space for the application data. While this data is not critical and thus can be lost, it is still needed for the application to function properly during its lifetime. This is where an `emptyDisk` can be useful. An emptyDisk is often used and mounted somewhere in `/var/lib` or `/var/run`.
 
+### configMap
+
+A `configMap` is a reference to a [ConfigMap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/) in Kubernetes. An extra `iso` disk will be allocated which has to be mounted on a VM. To mount the `configMap` users can use a `cloudInit` and disk serial number. The `configMapName` needs to be specified for a reference to the created kubernetes `ConfigMap` with the same name.
```

Comment:
`configMapName` is now just `name` I think. Maybe make the last sentence a little bit simpler.

Reply:
aah yeah, i just forgot about that, sorry, Changing...

Reply:
Maybe the last sentence could be like this:

The `name` has to be set for a reference to the created Kubernetes `ConfigMap`.

Sounds better and it's clear, what do you think?

Reply:
@stu-gott ^, is it ok? Any comment?